contrib/macdeploy/macdeployqtplus

```diff
@@ -302,7 +302,6 @@ def copyFramework(framework, path, verbose):
         if os.path.exists(fromContentsDir):
             toContentsDir = os.path.join(path, framework.destinationVersionContentsDirectory)
             shutil.copytree(fromContentsDir, toContentsDir, symlinks=True)
-            contentslinkfrom = os.path.join(path, framework.destinationContentsDirectory)
```

Comment:
What's the reason to remove this line? We don't use dynamic linked custom Frameworks in the Bitcoin-Qt.app bundle.